kubernetes/iter8.go

```diff
@@ -17,61 +17,113 @@ var iter8typeMeta = meta_v1.TypeMeta{
 	APIVersion: ApiIter8Version,
 }
 
-// Linked with https://github.com/iter8-tools/iter8-controller/blob/master/pkg/apis/iter8/v1alpha1/experiment_types.go
+// ExperimentSpec defines the desired state of Experiment
```

Comment:
It would be nice to keep in a comment the source of the ExperimentSpec, in that case future maintainers can easily jump there for changes.

Reply:
Add to link to v1alph2